recipe/meta.yaml

```diff
@@ -104,7 +104,8 @@ outputs:
         - starlette
         - toolz
         - uvicorn
-        - watchgod
```

Comment:
This is not longer used; `watchfiles` (same author, new project) is used instead.